skops/cli/_utils.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+
+def get_log_level(level: int = 0):
+    """Takes in verbosity from a CLI entrypoint (number of times -v specified),
+    and sets the logger to the required log level"""
+
+    all_levels = [logging.WARNING, logging.INFO, logging.DEBUG]
```

Comment:
IIUC, warning level logs would always be shown, even if I don't add `-v`. IMO, this should not be the case. So I would prefer that without `-v`, the log level should be `ERROR` or `CRITICAL`.

Reply:
From the python docs:

```The default level is WARNING, which means that only events of this level and above will be tracked, unless the logging package is configured to do otherwise.```

I followed this because it's the default for python packages. 

We can adjust the default level for our CLI, or we could also add a `-q, --quiet` option to reduce the log level

Reply:
I agree this is true for logging in general, but for CLIs, there is the convention to not print anything if there is no issue. Ideally, I would like to avoid having a `--quiet` option, because it increases complexity and because it's unclear how it interacts with `-v`.

Reply:
Hmm, good point.

I would raise that it *could* be considered an issue if untrusted types were present.

If a user had a file with untrusted types in it, and we converted it silently, they wouldn't know they needed to specify anything as trusted when using it.

Reply:
IMO, this is more of an issue for the person loading the file, and they will know (unless they use `trusted=True`). Personally, I would not report it by default, but I can live with the current way.